src/components/structures/TimelinePanel.js

```diff
@@ -245,6 +245,69 @@ var TimelinePanel = React.createClass({
         }
     },
 
+    _unpaginateEvents: function(backwards) {
+        // Do not unpaginate below timelineCap
```

Comment:
well, why not?

(anyway: won't this mean that we never unpaginate, because the timelinewindow ensures that we never have more than timelineCap events?)